frontend/src/container/MetricsApplication/Tabs/External/Table/reduceTo.ts

```diff
@@ -0,0 +1,111 @@
+export type ReduceTo =
+	| 'avg'
+	| 'min'
+	| 'max'
+	| 'p50'
+	| 'p90'
+	| 'p95'
+	| 'p99'
+	| 'latest';
+
+function min(ns: number[]): number {
+	let result = ns[0];
+
+	for (let i = 0; i < ns.length; i += 1) {
+		if (ns[i] < result) {
+			result = ns[i];
+		}
+	}
+
+	return result;
+}
+
+function max(ns: number[]): number {
```

Comment:
let's write test case also

Reply:
Fixed

Reply:
NOTE: I used math.js `quantileSeq` function as a test reference for `quantile`. 

https://mathjs.org/

<img width="973" alt="Screenshot 2023-04-28 at 2 33 30 PM" src="https://user-images.githubusercontent.com/9302460/235148440-458e7032-8dd7-40a4-9db6-3b7ba39e3b22.png">